tests/lib/external/snapd-testing-tools/tools/os.query

```diff
@@ -73,19 +73,59 @@ is_core24() {
 }
 
 is_core_gt() {
-    compare_ubuntu "${1:-}" "-gt"
+    VERSION=$1
```

Comment:
```suggestion
    VERSION="$1"
```

It this shellcheck clean?

Reply:
Could this be `local`, otherwise we clobber `VERSION`.

Reply:
fixed